### ISSUE TITLE
Tips and Tricks for Deep Learning

### ISSUE THREAD

**Issue by Bayler** [section-io/engineering-education#602]
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Deep Learning is a sub-field of ML inspired by the structure of our human brain. It uses neural networks to identify patterns within unstructured data.
These tips and tricks in deep learning are essential for improving the accuracy of deep learning networks.

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

In this article, we will discuss the tips and tricks essential for deep learning in:

1. Data processing
2. Training a neural network
3. Parameter tuning
4. Regularization
5. Good practices

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

**Comment by ninjaginja** [comment]
@Bayler Are you able to expand upon what the tips and tricks will entail? We've already had several articles cover the topic of deep learning, and I want to be sure that this content builds upon that without being repetitive.

**Comment by Bayler** [comment]
Sure. @ninjaginja this article covers techniques used to improve the accuracy of results in deep learning.

For example:
On regularization, I'll be touching on techniques such as Dropout and Early stopping.

On Parameter tuning, I'll be covering on:

1.  Weight Initialization
Here, I'll cover on the Xavier initialization technique.

Basically, this article touches on tips we can improve on training and accuracy of results 
2.  Optimizing convergence
-Fixed and adaptive learning rates such as Adam, RMSprop and Momentum

**Comment by ninjaginja** [comment]
Thanks @Bayler ! This is very helpful for us to evaluate the suitability of the topic. Based on your expanded description, I think this sounds like something our readers could really benefit from. This topic is approved.

**Comment by Bayler** [comment]
Thank you @ninjaginja
